### DISCUSSION TITLE
Game genie support

### DISCUSSION THREAD

**Discussion by AmogusNes** [btimofeev/UniPatcher#60] [Ideas]
Я придумал идею которая будет отличать UniPatcher от других патчеров
Идея: Создать аналог Game Genie Guy для андроид
Game Genie Guy позволяет применять game Genie коды к играм
(Для нес, и других консолей)

Надеюсь заметишь

**Comment by btimofeev** [comment]
Привет. Спасибо за идею. Это не сложно сделать, я когда-то давно делал такую прогу для symbian. Надо только время найти :)
